src/orchard/java/resource.clj

```diff
@@ -0,0 +1,52 @@
+(ns orchard.java.resource
+  "Resolve JVM resource-related paths."
+  {:added "0.5.0"}
+  (:require
+   [clojure.java.io :as io]
+   [orchard.classpath :as cp]
+   [orchard.misc :as u]))
+
+(defn- trim-leading-separator
+  [s]
+  (if (.startsWith s java.io.File/separator)
+    (subs s 1)
+    s))
+
+(defn get-project-resources
+  []
+  (mapcat
+   (fn [directory]
+     (->> directory
+          (file-seq)
+          (filter (memfn isFile))
+          (map (fn [file]
+                 (let [relpath (-> file
```

Comment:
FWIW this reimplements [`java.net.URI/relativize`][1]

[1]: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/URI.html#relativize(java.net.URI)

Reply:
Is it the exact copy? Is it safe to replace it, meaning, do we have tests against this? :smile: Will investigate